scipy/stats/stats.py

```diff
@@ -6655,18 +6655,18 @@ def chisquare(f_obs, f_exp=None, ddof=0, axis=0):
     See Also
     --------
     scipy.stats.power_divergence
-    scipy.stats.fisher_exact : A more powerful alternative to the chisquare
-                               test if any of the frequencies are less than 5.
+    scipy.stats.fisher_exact
+    scipy.stats.barnard_exact: An unconditional exact test. An alternative
+        to chi-squared test for small sample sizes.
```

Comment:
We should still have a description and there is a space between `:`. Is this ok @josef-pkt or we should write more? This description is the same as in Barnard's function.
```suggestion
    scipy.stats.fisher_exact : Fisher exact test on a 2x2 contingency table.
    scipy.stats.barnard_exact : An unconditional exact test. An alternative
        to chi-squared test for small sample sizes.
```